### ISSUE TITLE
Can I use it for real-time?

### ISSUE THREAD

**Issue by prnvjb** [nwojke/deep_sort#82]
Hi,
Thanks for this amazing tracking algorithm. 
I want to ask you about this tracking algorithm to use in real time video sequence(like camera). Can I use it to track detected objects(from yolo or ssd) in pedestrain case and evaluate new objects?

Any example is appreciated.

**Comment by omarabid59** [comment]
Yes, you can indeed. I'll be posting up sample code on my github within the next 24 hours, so check it out after that time period.

Note that I am running an SSD model trained on the COCO dataset with an NVIDIA GTX 1070.
